### ISSUE TITLE
It is not possible to create a page "Website root"

### ISSUE THREAD

**Issue by netzarbeiter** [contao/contao#5110]
### Affected version(s)

Contao 5.0.0-RC3

### Description

How can the error be reproduced:
- Setup a fresh Contao installation
- Add a new Page of type "Website root"
- Enable Language fallback

Stack Trace:
```
TypeError:
Contao\CoreBundle\Slug\Slug::generate(): Argument #2 ($options) must be of type iterable|int, string given, called in /home/…/public_html/website/vendor/contao/core-bundle/src/EventListener/DataContainer/PageUrlListener.php on line 89

  at vendor/contao/core-bundle/src/Slug/Slug.php:32
  at Contao\CoreBundle\Slug\Slug->generate()
     (vendor/contao/core-bundle/src/EventListener/DataContainer/PageUrlListener.php:89)
  at Contao\CoreBundle\EventListener\DataContainer\PageUrlListener->generateAlias()
     (vendor/contao/core-bundle/contao/drivers/DC_Table.php:3142)
  at Contao\DC_Table->save()
     (vendor/contao/core-bundle/contao/classes/DataContainer.php:487)
  at Contao\DataContainer->row()
     (vendor/contao/core-bundle/contao/drivers/DC_Table.php:2015)
  at Contao\DC_Table->edit()
     (vendor/contao/core-bundle/contao/classes/Backend.php:546)
  at Contao\Backend->getBackendModule()
     (vendor/contao/core-bundle/contao/controllers/BackendMain.php:149)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:44)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:152)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:74)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (public/index.php:44)   
```

**Comment by jochi44** [comment]
Additional info:

If there already is a page of type "Website root", it is not possible to create a page of type "Regular page". The Page type "Regular page" is not selectable in the "Page type" selection.

**Comment by fritzmg** [comment]
> If there already is a page of type "Website root", it is not possible to create a page of type "Regular page". The Page type "Regular page" is not selectable in the "Page type" selection.

Are you sure you did not accidentally try to add another website root? The page type selection will be limited to website roots at the root level.

**Comment by jochi44** [comment]
> > If there already is a page of type "Website root", it is not possible to create a page of type "Regular page". The Page type "Regular page" is not selectable in the "Page type" selection.
> 
> Are you sure you did not accidentally try to add another website root? The page type selection will be limited to website roots at the root level.

Sorry, my fault. I tried to create a new "Regular page" within the "top of the page tree".

**Comment by ausi** [comment]
Should be fixed by #5144
